src/RcppExports.cpp

```diff
@@ -390,7 +390,7 @@ BEGIN_RCPP
 END_RCPP
 }
 
-RcppExport SEXP run_testthat_tests(SEXP use_xml_sxp);
+RcppExport SEXP run_testthat_tests();
```

Comment:
Have you tried running the tests or doesn't that break CI tests? The call of some `testthat` internal function has changed but still was rendered incorrectly by Rcpp, that's why there is a difference in this parameter -- I've inserted that manually after generating `RcppExports.cpp`

Reply:
Everything seem to work now, so this workaround is no more necessary.